docs/Product/DataAnalysisValidation.mdx

```diff
@@ -1,27 +1,29 @@
 ---
 sidebar_position: 10.11
-sidebar_label: "Data analysis and validation"
+sidebar_label: "Data Analysis and Validation"
 ---
 
 import MaintainerTag from "@site/src/components/MaintainerTag";
 import ControlledDocBanner from "@site/src/components/ControlledDocBanner";
 
 <MaintainerTag maintainerEmails={["mario.dzamarija@openenergytransition.org", "fabrizio.finozzi@openenergytransition.org"]} />
 
-# Data analysis and validation
+# Data Analysis and Validation
 <ControlledDocBanner />
 
-## Data analysis
+## Data Analysis
```

Comment:
Please leave Data analysis. As highlighted here, we should use sentence case for titles https://open-energy-transition.github.io/handbook/docs/Handbook/HandbookStyleGuide

Reply:
Also corrected.